frontend/app/routes/_gcweb-app.data-unavailable.tsx

```diff
@@ -29,8 +29,31 @@ export default function DataUnavailable() {
   const { SCCH_BASE_URI } = getClientEnv();
   return (
     <>
-      <p>{t('service-unavailable')}</p>
-      <InlineLink to={t('gcweb:header.menu-dashboard.href', { baseUri: SCCH_BASE_URI })}>{t('gcweb:header.menu-dashboard.text')}</InlineLink>
+      <p className="mb-6">
+        {t('service-eligible')}
+        <InlineLink id="do-you-qualify-link" to={t('do-you-qualify.href')}>
+          {t('do-you-qualify.text')}
+        </InlineLink>
+        .
+      </p>
+      <p className="mb-6">
+        {t('more-information')}
+        <InlineLink id="how-to-apply-link" to={t('how-to-apply.href')}>
+          {t('how-to-apply.text')}
+        </InlineLink>
+        .
+      </p>
```

Comment:
You should use the `<Trans>` component here instead of breaking the i18n into multiple chunks.